### ISSUE TITLE
Take a picture when a wrong code is entered

### ISSUE THREAD

**Issue by giuseppeg88** [thanksmister/android-mqtt-alarm-panel#112]
**Describe the solution you'd like**
It would be nice to have the ability to automatically take a picture when a wrong code is entered; the picture could then be sent via the options already present in the app.

Furthermore, it would be wonderful if this functionality could be enabled even without having to keep the camera stream active (which makes the phone heat up).

**Comment by thanksmister** [comment]
I think this feature should be a remote command, to take a photo and send it using MQTT or HTTP.

**Comment by giuseppeg88** [comment]
In the standard configuration, if an incorrect code is entered, no mqtt message is transmitted making it impossible to detect the event to which to associate the photo taking. It would be more immediate to ask the app to take the photo in case of wrong password.

Also, in the current release, it is not possible to take a photo without first enabling the video stream. It would be great not to be forced to enable the video stream (which warms up the phone) to take photos.

Thanks for your reply.

**Comment by thanksmister** [comment]
Incorrect code does return feedback is using alarmó integration or custom MQTT which passes code to server.  

I think this is better solution to leverage server and MQTT commands than add feature to alarm, but I’m open to a pull request with either solution.

My reasoning is that I do not like to depend on another api or service to send images, ideally would be job of server not client.

**Comment by giuseppeg88** [comment]
Good idea! I like the idea of using the new way of passing code to the server, which allows for more control and feedback.

Regarding the order to be sent, is it possible to take a photo without being forced to enable the video stream (which warms up the phone)?
